pkg/convert/pot.go

```diff
@@ -73,46 +51,124 @@ func FromPot(b []byte) (model.Messages, error) {
 	}
 
 	messages := make([]model.Message, 0, len(po.Messages))
-	// model.Messages does not support plurals atm. But we plan to impl it under:
-	// https://github.com/orgs/translate-agent/projects/1?pane=issue&itemId=21251425
+
 	for _, node := range po.Messages {
-		if strings.HasSuffix(node.MsgId, "\\n") {
-			node.MsgId = strings.ReplaceAll(node.MsgId, "\\n", "\n")
+		message := model.Message{ID: node.MsgId}
+
+		switch {
+		case po.Header.PluralForms.NPlurals > pluralCountLimit:
+			return model.Messages{}, fmt.Errorf("plural forms with more than 2 forms are not implemented yet")
+		case po.Header.PluralForms.NPlurals > 1:
+			message.PluralID = node.MsgIdPlural
+			message.Message = convertToFormatMessage(node.MsgStr)
+		default:
+			message.Message = node.MsgStr[0]
```

Comment:
```suggestion
	for _, node := range po.Messages {
		message := model.Message{ID: node.MsgId, PluralID: node.MsgIdPlural}

		switch {
		case po.Header.PluralForms.NPlurals > pluralCountLimit:
			return model.Messages{}, fmt.Errorf("plural forms with more than 2 forms are not implemented yet")
		case po.Header.PluralForms.NPlurals > 1:
			message.Message = convertToFormatMessage(node.MsgStr)
		default:
			message.Message = node.MsgStr[0]
		}
```

You could define message with on 56 line with `MsgIDPLural`, if I am not mistaken when `po.Header.PluralForms.NPlurals` is 1, then `MsgIDPLural` is empty string.
If not nevermind